README.md

```diff
@@ -8,24 +8,14 @@ This action is built on top of [VCS Diff Lint](https://github.com/fedora-copr/vc
 name: Differential Linters
 
 on:
+  push:
   pull_request:
     branches: [ main ]
 
-permissions:
-  contents: read
-
 jobs:
   lint:
     runs-on: ubuntu-latest
-
-    permissions:
-      # required for all workflows
-      security-events: write
```

Comment:
`security-events: write` are required by `github/codeql-action/upload-sarif`. It might work without it, but only for repositories that don't harden GITHUB_TOKEN permissions.

![Screenshot from 2023-04-05 09-49-48](https://user-images.githubusercontent.com/2879818/230016123-805a4732-2351-47bf-80da-f90901317663.png)

Reply:
Ah, thank you for the note!  Then we need this.